docs/guide/options/README.md

```diff
@@ -485,7 +483,8 @@ messaging: true
 // or
 
 messaging: {
-  createServiceWorker: false
+  createServiceWorker: false,
+  fcmPublicVapidKey: '<publicVapidKey>' // OPTIONAL : Sets vapid key for FCM after initialization
```

Comment:
here should also go the `notificationKey` and the `notificationOptionsTemplate `